Documentation/customresourcedefinition-metrics.md

```diff
@@ -0,0 +1,10 @@
+# Stateful Set Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| kube_customresourcedefinition_created | Gauge | `customresourcedefinition`=&lt;customresourcedefinition-name&gt; | EXPERIMENTAL |
+| kube_customresourcedefinition_labels | Gauge | `customresourcedefinition`=&lt;customresourcedefinition-name&gt; <br> `label_CRD_LABEL`=&lt;CRD_LABEL&gt;  | EXPERIMENTAL |
```

Comment:
is this actually useful? I've never seen anyone label a CustomResourceDefinition

Reply:
https://github.com/kubernetes/cluster-registry/blob/master/cluster-registry-crd.yaml#L7

Reply:
fair :+1: